src/integrationTest/java/com/pubnub/api/integration/PublishIntegrationTests.java

```diff
@@ -516,6 +519,8 @@ public void status(@NotNull PubNub pubnub, @NotNull PNStatus pnStatus) {
             @Override
             public void message(@NotNull PubNub pubnub, @NotNull PNMessageResult pnMessageResult) {
                 final JsonElement receivedMessage = pnMessageResult.getMessage();
+                assertEquals(PNMessageType.MESSAGE01.toString(), pnMessageResult.getMessageType().getValue());
+                assertEquals(PNMessageType.MESSAGE02.toString(), pnMessageResult.getMessageType().getValue());
```

Comment:
Why two asserts? If you want to make sure that `MESSAGE01` and `MESSAGE02` have same value then it's not the best place for this

Reply:
Initially I had only one assertion like:
`assertEquals(PNMessageType.MESSAGE01.toString(), pnMessageResult.getMessageType().getValue());`
but then I though that it is misleading because also second case is valid.

What I can do is delete those two assertions and create simple test like:

```
class PNMessageTypeTest {

    @Test
    void message_of_type_message_should_be_indicated_by_messageType_being_null_or_0() {
        assertEquals(PNMessageType.MESSAGE01.toString(), PNMessageType.MESSAGE02.toString());
    }
}
```

What do you think?